app/form_models/jobseekers/job_application/professional_status_form.rb

```diff
@@ -45,16 +45,30 @@ def initialize(attributes = {})
         self.has_teacher_reference_number ||= jobseeker_profile.has_teacher_reference_number
       end
 
-      validates :qualified_teacher_status, inclusion: { in: %w[yes no on_track] }, if: -> { professional_status_section_completed }
-      validates :qualified_teacher_status_year, numericality: { less_than_or_equal_to: proc { Time.current.year } },
-                                                if: -> { qualified_teacher_status == "yes" && professional_status_section_completed }
-      validates :statutory_induction_complete, inclusion: { in: %w[yes no] }, if: -> { professional_status_section_completed }
-
-      validates :teacher_reference_number, presence: true, if: -> { qualified_teacher_status == "yes" && professional_status_section_completed }
-      validates_format_of :teacher_reference_number, with: /\A\d{7}\z/, allow_blank: false, if: -> { qualified_teacher_status == "yes" || has_teacher_reference_number == "yes" }
-      validates_format_of :teacher_reference_number, with: /\A\d{7}\z/, allow_blank: true, if: -> { qualified_teacher_status.in?(%w[no on_track]) }
-      validates :has_teacher_reference_number, inclusion: { in: %w[yes] }, if: -> { qualified_teacher_status == "yes" && professional_status_section_completed }
-      validates :has_teacher_reference_number, inclusion: { in: %w[yes no] }, if: -> { qualified_teacher_status.in?(%w[no on_track]) && professional_status_section_completed }
+      # These validations are only applied when the professional status section is marked as completed.
+      #
+      # Equivalent to if: -> { professional_status_section_completed == true }.
+      # Nested validations using "if" calls would override the parent "with_options if:" call instead of combining the conditions.
+      # By using "unless" we can keep using "if" for nested validations and the "if & unless" conditions will be combined.
+      with_options unless: -> { professional_status_section_completed != true } do
```

Comment:
So we can't use `if` here and also use `if` for nested validations?

Reply:
Exactly, that is what I was trying to alert about with the comment:
```
with_options if: -> { condition A }
  validate .... if: -> { condition B }
end
```
The output for this would be `  validate .... if: -> { condition B }` as the nested `if` overrides the parent.
Instead
```
with_options unless: -> { condition A }
  validate .... if: -> { condition B }
end
```
The output for this is `validate .... if: -> { condition B }, unless:  -> { condition A }`

Reply:
Ah I did see that from the comment but it looked so weird that I wanted to double check I understood, this makes a lot of sense: `The output for this is validate .... if: -> { condition B }, unless:  -> { !condition A }.`

Thanks for the great explanation 🙇